UndertaleModTool/MainWindow.xaml.cs

```diff
@@ -1662,22 +1851,22 @@ public void SetProgressBar()
             if (scriptDialog != null && !scriptDialog.IsVisible)
                 scriptDialog.Dispatcher.Invoke(scriptDialog.Show);
         }
+
         public void UpdateProgressValue(double progressValue)
         {
             if (scriptDialog != null)
             {
-                scriptDialog.Dispatcher.Invoke(DispatcherPriority.Normal, (Action)(() => {
-                    scriptDialog.ReportProgress(progressValue);
-                }));
+                scriptDialog.Dispatcher.Invoke(DispatcherPriority.Normal, (Action) (() => { scriptDialog.ReportProgress(progressValue); }));
```

Comment:
It will be better to `scriptDialog.ReportProgress(progressValue);` be on separate line, as it was before.

Reply:
Was done by auto-formatting, didn't pay attention to it